pkgs/development/python-modules/pypdfium2/default.nix

```diff
@@ -9,15 +9,14 @@
   numpy,
   pillow,
   pytestCheckHook,
-  python,
 }:
 
 let
   pdfiumVersion = "${pdfium-binaries.version}";
 
   headers = fetchurl {
     url = "https://pdfium.googlesource.com/pdfium/+archive/refs/heads/chromium/${pdfiumVersion}/public.tar.gz";
-    hash = "sha256-vKfs4Jd8LEtA3aTI+DcJMS0VOErq1IR1eThnMlxiER0=";
+    hash = "sha256-RJOhv9v/uI8jYh0hoM7FxGu2lcVV+7BjLkiZW0eX4hk=";
```

Comment:
This isn't even the hash obtained in https://github.com/NixOS/nixpkgs/issues/378154.

Reply:
I wrote #378154 using x86-Linux but was developing this PR on aarch64-darwin. (The linux box was bust running a nixpkgs-review.)